### DISCUSSION TITLE
Add Input to Molecule Select

### DISCUSSION THREAD

**Discussion by Alverd04** [SUI-Components/sui-components#2220] [Ideas and new features]
Hi, I've been using coches.net today and I just noticed that whenever you want to select a model or brand you have to scroll till you find it. That ended up giving me a UX improvement, that could be adding a input to let the user write and filter the list suggestions and values. If you think it's a good idea I can submit a pull request showing how it would work!

**Comment by turolopezsanabria** [comment]
Oh!! 
this is a good one!

Do you think it might be related to this Jira ticket? (image attached)
https://jira.scmspain.com/browse/DO-555

![combobox](https://user-images.githubusercontent.com/23620759/176453720-537993a6-b07d-441b-a8c0-fe9f5adf5256.png)

**Reply by Alverd04** [reply]
Exactly!!!
